src/features/user/forms/PasswordForm.js

```diff
@@ -14,18 +14,28 @@ export const PasswordForm = ({ Component, title, onDone }) => {
     // 2. Listen for those PasSwords from those inputs
     const oldPassword = event.target.elements.currentPasswordInput.value
     const newPassword = event.target.elements.newPasswordInput.value
-
+    const newSamePassword = event.target.elements.newSamePassword.value
+    // Before calling updateUser make sure the password in both of the new password inputs are the same
     // 3. Do somethings like, send those PasSwords to Daniel's and  ...'s Userbase
-    const { error } = await updateUser({
-      currentPassword: oldPassword,
-      newPassword: newPassword,
-    })
 
-    // 4. Send that customer back to /profile or give alert if error
-    if (error) {
-      alert(error.message)
+    if (newPassword !== newSamePassword) {
+        alert(
+          `You typed your "New Password Again" incorrectly, please try again.`
+        )
     } else {
-      onDone()
+      const { error } = await updateUser({
+        currentPassword: oldPassword,
+        newPassword: newPassword,
+      })
+
+      alert(`Success, your new password is good to go.`)
```

Comment:
**suggestion:** Move this alert into the else below, otherwise it pops up even if updateUser fails.

To see why test by typing a wrong current password, but matching new and new again passwords.

Reply:
Move this alert into the else below #276
94fb1d3